app.py

```diff
@@ -158,22 +179,20 @@ def _side_effects(source, new_commit):
     if not source["name"] == QUIPUCORDS_SERVER:
         return
     _update_quipucords_sha(new_commit)
-    _handle_cryptography(source, new_commit)
+    _handle_rust_deps(source, new_commit)
 
 
-def _handle_cryptography(source, new_commit):
-    console.print(
-        "Checking if quipucords subdependency 'cryptography' needs to be updated."
-    )
+def _handle_rust_deps(source, new_commit):
```

Comment:
I don't think handle_rust is a very descriptive name. Maybe you could use something like check_rust_updates.